package.json

```diff
@@ -54,8 +56,9 @@
     "@nuxt/test-utils": "3.11.0",
     "@nuxtjs/i18n": "8.2.0",
     "@nuxtjs/sitemap": "5.1.2",
+    "@rollup/rollup-linux-arm64-gnu": "4.13.0",
```

Comment:
is there any reason why we need to specify this package? seeing that it is architecture/os specific?

Reply:
yeah I was just trying things.
So there is some issue with --no-optional and rollup
https://github.com/npm/cli/issues/4828 the answer is here

Reply:
updated